README.md

```diff
@@ -2,182 +2,232 @@
 
 This library allows you to safely test your migration from one back-end to another in production!
 
-The Shadow Tool can be easily integrated in your Java/Kotlin project and allows you to compare the current back-end
+The Shadow Tool can be easily integrated into your Java/Kotlin project and allows you to compare the current back-end
 service your application is using against the new back-end you plan on using.
-Since it actually runs on production (in the background), it gives you trust in:
+Since it actually runs in the production environment (in the background), it helps ensure that:
 
 1. the connection towards your new back-end,
 2. the data quality coming from the new back-end,
 3. whether your code correctly maps the data of the new back-end to your existing domain.
```

Comment:
These sentences aren't finished now changed it into "helps ensure that".

Reply:
what about `it helps **ensuring** that:`